.github/release-drafter-config.yml

```diff
@@ -0,0 +1,23 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+template: |
+  <!-- Optional: add a release summary here -->
+  $CHANGES
+
+name-template: $NEXT_MINOR_VERSION
+tag-template: $NEXT_MINOR_VERSION
+# Uses a more common 2-digit versioning in Jenkins plugins. Can be replaced by semver: $MAJOR.$MINOR.$PATCH
+version-template: $MAJOR.$MINOR
+
+# Emoji reference: https://gitmoji.carloscuesta.me/
+categories:
+  - title: "Breaking changes"
+    label: "breaking change"
+
+  - title: "Features"
+    label: "feature"
+
+  - title: "Enhancements"
+    label: "enhancements"
+
+  - title: "Bug Fixes"
+    labels: "bug fix"
```

Comment:
Can you verify these are the same labels we have?

Reply:
Done